src/Css.elm

```diff
@@ -974,8 +975,7 @@ combineLengths operation firstLength secondLength =
             [ String.fromFloat numericValue
             , firstLength.unitLabel
             ]
-                |> List.filter (not << String.isEmpty)
-                |> String.join ""
+                |> String.filterJoin (not << String.isEmpty) ""
```

Comment:
I'm squinting at this, but why is the filter even necessary here? Wouldn't joining empty strings just end up being exactly the same? Is that a perf optimization in itself?

Reply:
That's a very good point!

Reply:
Yeah it looks like this whole thing can be `String.fromFloat numericValue ++ firstLength.unitLabel`.

I suspect the reason it wasn't in the first place is that it was copy/pasted and modified from something that was joining with newlines (instead of `""`), where doing the filter actually meant not printing unnecessary blank lines.

Reply:
I think between this change and https://github.com/rtfeldman/elm-css/pull/544/files#r744673432 we don't need `filterJoin` and can take it out of the PR! 😄